From 25878e4b1b4c449e3f21f9cc536c3e3b84af8ad8 Mon Sep 17 00:00:00 2001
From: Alexander Shiyan <shc_work@mail.ru>
Date: Sat, 20 Jul 2013 11:17:56 +0400
Subject: [PATCH 125/161] ARM: dts: i.MX27: Add label to CPU node

Add a label to i.MX27 CPU node. This change allows the reuse this node
in the upper levels of the DTS.

Signed-off-by: Alexander Shiyan <shc_work@mail.ru>
Acked-by: Sascha Hauer <s.hauer@pengutronix.de>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/boot/dts/imx27.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx27.dtsi b/arch/arm/boot/dts/imx27.dtsi
index 7cf954b..a8e0c7e 100644
--- a/arch/arm/boot/dts/imx27.dtsi
+++ b/arch/arm/boot/dts/imx27.dtsi
@@ -53,7 +53,7 @@
 		#size-cells = <0>;
 		#address-cells = <1>;
 
-		cpu {
+		cpu: cpu@0 {
 			device_type = "cpu";
 			compatible = "arm,arm926ej-s";
 			operating-points = <
-- 
1.7.10.4

